packages/institutions/src/components/Institutions/institution.jsx

```diff
@@ -0,0 +1,109 @@
+import {
+  Box,
+  Divider,
+  Grid,
+  List,
+  ListItem,
+  ListItemText,
+  Typography,
+} from '@mui/material';
+import React from 'react';
+import Image from 'next/image';
+import { ButtonStyled } from '@siiges-ui/shared';
+
+export default function Institution() {
+  return (
+    <Grid container>
```

Comment:
Please change the name of the file to index.jsx
Please rename the directory like "Institucion" "InstitucionLayout"
And to components inside "InstitucionData" and "PlantelesList"
And inside "PlantelesList" a component like "PlantelData" to add or edit platel